kelvin-calculator/src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component, useEffect, useState } from 'react'
+import {evaluate} from "mathjs";
+import NumberFormat from 'react-number-format'
+import '../components/Calculator.css'
+
+export default class Calculator extends Component {
+
+
+    
+        state = {
+          input: ""
+        };
+      
+
+    inputChange = (e) => {
+        this.setState({input: this.state.input + e.target.value});
```

Comment:
if you had the two states this would something like `this.state.input` +`this.state.displayVal`